gdx/src/com/badlogic/gdx/math/Intersector.java

```diff
@@ -285,7 +304,8 @@ public static Vector2 nearestSegmentPoint (float startX, float startY, float end
 		return nearest.set(startX + t * (endX - startX), startY + t * (endY - startY));
 	}
 
-	/** Returns whether the given line segment intersects the given circle.
+	/** Checks whether the given line segment intersects the given circle.
```

Comment:
Starting javadoc with "Returns ..." is a common pattern and I think more descriptive than "Checks ..." (which is see is ued elsewhere in the file). Also by using returns in the text, an @return is not needed at all.